rtos/mbed_rtos_types.h

```diff
@@ -58,6 +58,62 @@ typedef enum {
+    osPriorityBelowNormal1  = 16 + 1,     ///< Priority: below normal + 1
+    osPriorityBelowNormal2  = 16 + 2,     ///< Priority: below normal + 2
+    osPriorityBelowNormal3  = 16 + 3,     ///< Priority: below normal + 3
+    osPriorityBelowNormal4  = 16 + 4,     ///< Priority: below normal + 4
+    osPriorityBelowNormal5  = 16 + 5,     ///< Priority: below normal + 5
+    osPriorityBelowNormal6  = 16 + 6,     ///< Priority: below normal + 6
+    osPriorityBelowNormal7  = 16 + 7,     ///< Priority: below normal + 7
+    osPriorityNormal        = 24,         ///< Priority: normal
+    osPriorityNormal1       = 24 + 1,     ///< Priority: normal + 1
+    osPriorityNormal2       = 24 + 2,     ///< Priority: normal + 2
+    osPriorityNormal3       = 24 + 3,     ///< Priority: normal + 3
+    osPriorityNormal4       = 24 + 4,     ///< Priority: normal + 4
+    osPriorityNormal5       = 24 + 5,     ///< Priority: normal + 5
+    osPriorityNormal6       = 24 + 6,     ///< Priority: normal + 6
+    osPriorityNormal7       = 24 + 7,     ///< Priority: normal + 7
+    osPriorityAboveNormal   = 32,         ///< Priority: above normal
+    osPriorityAboveNormal1  = 32 + 1,     ///< Priority: above normal + 1
+    osPriorityAboveNormal2  = 32 + 2,     ///< Priority: above normal + 2
+    osPriorityAboveNormal3  = 32 + 3,     ///< Priority: above normal + 3
+    osPriorityAboveNormal4  = 32 + 4,     ///< Priority: above normal + 4
+    osPriorityAboveNormal5  = 32 + 5,     ///< Priority: above normal + 5
+    osPriorityAboveNormal6  = 32 + 6,     ///< Priority: above normal + 6
+    osPriorityAboveNormal7  = 32 + 7,     ///< Priority: above normal + 7
+    osPriorityHigh          = 40,         ///< Priority: high
+    osPriorityHigh1         = 40 + 1,     ///< Priority: high + 1
+    osPriorityHigh2         = 40 + 2,     ///< Priority: high + 2
+    osPriorityHigh3         = 40 + 3,     ///< Priority: high + 3
+    osPriorityHigh4         = 40 + 4,     ///< Priority: high + 4
+    osPriorityHigh5         = 40 + 5,     ///< Priority: high + 5
+    osPriorityHigh6         = 40 + 6,     ///< Priority: high + 6
+    osPriorityHigh7         = 40 + 7,     ///< Priority: high + 7
+    osPriorityRealtime      = 48,         ///< Priority: realtime
+    osPriorityRealtime1     = 48 + 1,     ///< Priority: realtime + 1
+    osPriorityRealtime2     = 48 + 2,     ///< Priority: realtime + 2
+    osPriorityRealtime3     = 48 + 3,     ///< Priority: realtime + 3
+    osPriorityRealtime4     = 48 + 4,     ///< Priority: realtime + 4
+    osPriorityRealtime5     = 48 + 5,     ///< Priority: realtime + 5
+    osPriorityRealtime6     = 48 + 6,     ///< Priority: realtime + 6
+    osPriorityRealtime7     = 48 + 7,     ///< Priority: realtime + 7
+    osPriorityISR           = 56,         ///< Reserved for ISR deferred thread.
+    osPriorityError         = -1,         ///< System cannot determine priority or illegal priority.
```

Comment:
Do we really need all these other priorities? Or would just osPriorityError be fine?

Reply:
This is been taken from cmis_os2.h to match what we are talking about. `osStatus_t` is copied from there.